client/containers/Pub/PubDocument/PubFileImport/FileImportDialog.tsx

```diff
@@ -221,6 +228,18 @@ const FileImportDialog = ({
 									</details>
 								</li>
 							)}
+							{pandocInfos && (
+								<li>
+									<i>
+										Conversion from LaTeX to HTML succeeded, with information (
+										<a href="https://help.pubpub.org/pub/2gfph6h8/release/2">
```

Comment:
Perhaps we should not link to a specific release (and maybe also give this Pub a nice slug?)